docs/guides/creatingasplitter.md

```diff
@@ -1,17 +1,27 @@
-# Creating a Splitter
+# Creating an Event Splitter
 
-Splitter is a part of TriggerMesh routing solution that has a simple purpose to
-split JSON arrays into multiple CloudEvents.
+An event `Splitter` is part of the TriggerMesh routing solution. It has a simple purpose of splitting JSON arrays into multiple [CloudEvents](https://cloudevents.io/).
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md). You can verify that the API is available with the following command:
+    
+    ```console
+       $ kubectl get crd|grep splitter
+       splitters.routing.triggermesh.io                        2021-10-06T09:01:38Z
+    ```
```

Comment:
As discussed on Slack, I think this is super redundant ("Guess what, you need to run TriggerMesh to use TriggerMesh").
I'd be curious to know what others think, maybe I'm just picky, but I haven't seen that anywhere else.

Reply:
I do think that having a little tip up front that could do a sanity check to see if splitters are actually set up would be nice.

Reply:
Fair enough